src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/repository/PlayerRepository.java

```diff
@@ -0,0 +1,13 @@
+package ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.repository;
+
+import ch.uzh.ifi.hase.soprafs22.screwyourneighborserver.entity.Player;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource
+public interface PlayerRepository extends PagingAndSortingRepository<Player, Long> {
+  @SuppressWarnings("unused")
+  Page<Player> findAllByName(String name, Pageable page);
```

Comment:
If we need to filter a collection, we can write a method with a certain naming pattern, and we can then filter after this property